### PR TITLE
R4R: Add docs and test for HTLC

### PR DESCRIPTION
**Pull request by SegueII**
Resolves: #1965

### CODE REVIEW

File: docs/cli-client/htlc/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+```bash
+ iriscli htlc [command]
+```
+
+## Available Commands
+
+| Name                        | Description                 |
+| --------------------------- | --------------------------- |
+| [create](create.md)         | Create a HTLC               |
```
Comment (haifengxi) [inline docs/cli-client/htlc/README.md]: a HTLC --> an HTLC
a htlc --> an HTLC

Please make the changes throughout the document.

File: docs/cli-client/htlc/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# iriscli htlc
+
+## Description
+
+Htlc allows you to manage local htlc for atomic swap with other chains
```
Comment (haifengxi) [inline docs/cli-client/htlc/README.md]: ... to manage local Hash Time Locked Contracts (HTLCs) for atomic swaps ...

File: docs/cli-client/htlc/claim.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+```bash
+iriscli htlc claim --hash-lock=<hash-lock> --secret=<secret>
+```
+
+## Flags
+
+| Name, shorthand | Type     | Required | Default | Description                                     |
+| --------------- | -------- | -------- | ------- | ----------------------------------------------- |
+| --hash-lock     | bytesHex | true     |         | The hashLock identifying the htlc to be claimed |
```
Comment (haifengxi) [inline docs/cli-client/htlc/claim.md]: The hash lock identifying the HTLC to be claimed

File: docs/cli-client/htlc/claim.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+| --secret        | string   | true     |         | The secret for generating hashLock              |
```
Comment (haifengxi) [inline docs/cli-client/htlc/claim.md]: hashLock --> hash lock

File: lite/swagger-ui/swagger.yaml [hunk 5]
```diff
@@ -3065,7 +3062,7 @@ paths:
                   $ref: '#/components/schemas/Address'
                 secret:
                   type: string
-                  example: '327a66212cbee3756816e617b38f9dfe327a66212cbee3756816e617b38f9dfe'
+                  example: '___abcdefghijklmnopqrstuvwxyz___'
```
Comment (chengwenxi) [inline lite/swagger-ui/swagger.yaml]: should be hex encoded string.

**Pull request merged**